opus_test.go

```diff
@@ -91,6 +91,178 @@ func TestCodecFloat32(t *testing.T) {
 	}
 }
 
+func TestCodecFEC(t *testing.T) {
+	// Create bogus input sound
+	const G4 = 391.995
+	const SAMPLE_RATE = 48000
+	const FRAME_SIZE_MS = 60
+	const FRAME_SIZE = SAMPLE_RATE * FRAME_SIZE_MS / 1000
+	const PACKET_SIZE = 480 // 10ms packet
```

Comment:
This is in fact the frame, not the 60ms defined further up. Your entire audio stream is 60ms, the frame size is 10ms. Packets are more of a "transport level" term, when you start talking about networks. If you send every frame as a separate network message (which will typically be the case), then frame ~= packet. Either way, you'll want to use 10 as your FRAME_SIZE_MS, and use a different name for the 60 constant you defined above.

Reply:
Thanks for the explanation, I wasn't sure about that.